server/app/views/dev/DatabaseSeedView.java

```diff
@@ -52,8 +59,37 @@ public Content render(
     String prettyActivePrograms = getPrettyJson(activePrograms);
     String prettyQuestions = getPrettyJson(questionDefinitions);
 
+    ATag indexLinkTag =
+        a().withHref(routes.DevDatabaseSeedController.index().url())
+            .withId("index")
+            .with(submitButton("index", "Go to dev database seeder page"));
+
+    DivTag content =
+        div()
+            .with(h1(title))
+            .with(indexLinkTag)
+            .with(
+                form()
```

Comment:
**question (blocking):** what is this form element for?

Reply:
changed it to `div` since it's a get method